smoker.spec

```diff
@@ -1,8 +1,8 @@
 %global with_check 0
 
 Name:		smoker
-Version:	2.1.13
-Release:	1%{?dist}
+Version:	2.1.17
+Release:	2%{?dist}
```

Comment:
I wouldn't bump the release here, the `make version` target ignores it and it's completely invisible on the python egg versioning level.